juego.py

```diff
@@ -181,14 +199,13 @@ def play(self, level):
 
         another_quest = True
 
-        right_sound = load_sound("./data/right.ogg")
-        wrong_sound = load_sound("./data/wrong.ogg")
-        fondo = [load_image("data/" + str(x) + ".jpg") for x in range(0,NUM_IMAGES)]
-        random_index = random.randint(0,NUM_IMAGES - 1)
```

Comment:
random images no longer occur?

Reply:
There is just one image that needs to be loaded, which is stored as data/1.jpg
See closed issue #5